test/acceptance/features/bindAppToMultipleServices.feature

```diff
@@ -48,7 +48,60 @@ Feature: Bind a single application to multiple services
                     kind: Database
                     resourceRef: db-demo-2
             """
-        Then "nodejs-app" deployment must contain SBR name "binding-request-1" and "binding-request-2"
+        Then deployment must contain intermediate secret
```

Comment:
can we be more precise what does it? what does it "contain"? in which part? what "intermediate" does mean to users?

Reply:
Can be like this 
`Then "nodejs-app" deployment must contain intermediate secret`